### ISSUE TITLE
queryTests.php filters are not complete

### ISSUE THREAD

**Issue by mathstuf** [Kitware/CDash#478]
It seems that the filters in `queryTests.php` don't include filters such as `Revision` and many many others available from the main dashboard page.

**Comment by mathstuf** [comment]
@zackgalbreath FYI

**Comment by zackgalbreath** [comment]
I can see the benefit of having the Revision filter available on queryTests.php.

I'm less convinced that queryTests.php should contain all the same filters as index.php, though.  These pages display different data.  Also, the rows on index.php are builds, whereas the rows on queryTests.php are tests.

Currently queryTests.php allows you to filter based on the information that's displayed on the page: site, build name, test details, etc.  I worry that exposing filters like "number of build errors" or "update duration" would be confusing, since this information is not exposed (or really relevant) here.

**Comment by mathstuf** [comment]
Hmm, true. Yeah, the non-test step-related filters probably don't make sense.

**Comment by mathstuf** [comment]
Then again, maybe not supporting them in the dropdown works, but supporting them as filters may help (so that it's easy to search in the main page and then go and "see all test failures" for that filtered set).

**Comment by zackgalbreath** [comment]
> Then again, maybe not supporting them in the dropdown works, but supporting them as filters may help (so that it's easy to search in the main page and then go and "see all test failures" for that filtered set).

Good point.  Maybe it's worthwhile to faithfully reproduce this set after all.  I'll ponder it a bit and reply back here once I have something to share.
